modules/compute/eks/stack.py

```diff
@@ -477,6 +482,16 @@ def _create_eks_cluster(
         Creates an Amazon EKS cluster with the specified configuration.
         """
 
+        if eks_compute_config.get("eks_api_endpoint_private"):
+            api_endpoint = eks.EndpointAccess.PRIVATE
+        else:
+            api_endpoint = eks.EndpointAccess.PUBLIC_AND_PRIVATE
```

Comment:
What if the user is not passing in an SSM but the acutal CIDR's.  I think this locks the use case to only SSM parameters....

Reply:
Agreed, allow both

Reply:
Good point.